Update to Polymer 2.0 preview and fix tests (dash-35)

alvarezguille opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/license-checker/25)
<!-- Reviewable:end -->

alvarezguille committed:
Update to polymer 2

alvarezguille committed:
Fix tests

rogozinds commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 11 of 11 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/license-checker/25#-:-Ke2fPtYDyut5Ax9T27e:bnfp4nl)*
<!-- Sent from Reviewable.io -->

alvarezguille committed:
Change creation of dialog

alvarezguille committed:
Update polymer to rc

Peppe committed:
Added name to input field (#28)

Added name to input field in hopes that it will give the possibility to store the key in browser memory and thus suggest it the next time you see the popup. Doesn't help if you nuke the whole browser cache, but might help if you clear local storage specifically. And shouldn't hurt.

alvarezguille committed:
Add product headers to license check request (#30)

alvarezguille committed:
Merge branch 'master' into 2.0-preview

alvarezguille commented:
Reviewed 2 of 2 files at r2, 1 of 1 files at r3, 5 of 5 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/license-checker/25)*
<!-- Sent from Reviewable.io -->
